.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "mxnet"]
 	path = mxnet
-	url = https://github.com/zhreshold/mxnet
+	url = https://github.com/larroy/mxnet
```

Comment:
What's the point of update the submodule?

Reply:
I rebased against master, your mxnet subrepo is outdated and have python3 problems that are solved in more recent versions. Of course prior to merging you could also pull my rebase of your changes to mxnet with the Yolo C++ operators so your subrepo is up to date, and this points to your subrepo on the final merge.